src/components/GridDisplay.tsx

```diff
@@ -19,7 +19,7 @@ const GridDisplay = ({ activeLink, limit, handleLoadingMore, fetchedData, user }
       {fetchedData.length > 0 && activeLink !== "myVotes" && limit <= 100 && (
         <button
           onClick={() => handleLoadingMore()}
-          className="bg-grey hover:bg-lightGrey text-white font-bold py-2 px-4 rounded-xl"
+          className="bg-gray100 uppercase border-[1px] hover:text-white border-gray120 hover:bg-gray150 text-gray150 font-bold py-2 px-4 rounded-xl"
```

Comment:
I see that before it was hover:bg-lightGrey, is this really a desired change ? Or you want to did like this `hover:bg-gray-100`

Reply:
@NsdHSO It was supposed to be "grey" instead of "gray" everywhere as I had to change that spelling on the Tailwind Config file. It is fixed now. Thanks!